docs/store/downgrade.md

```diff
@@ -0,0 +1,35 @@
+# Using ngrx/store in AngularJS
```

Comment:
I think this section can be added to the beginning of the next section.

```
If you are working on an AngularJS to Angular conversion, you can use `@ngrx/store` to provide global state to your hybrid application.
```

Reply:
Sorry, I'm not sure I totally understand what you have in mind.
Do you want to remove the second title with its text?
```
## Downgrading Store service

If you want to **dispatch** an action or **select** some slice of your store
state, you will need to downgrade the Store service to use it in the AngularJS
parts of your application.
```

Reply:
Replace the text below the `Using ngrx/store in AngularJS` section with the text I mentioned. Leave the Downgrade text as-is.